Organization announcements (#94)
hackclub/hcb-engr: content/posts/organization-announcements/post.mdx

import post from '@/components/Post';
export default post({
  title: "📣 Announcing organization announcements!",
  slug: "organization-announcements",
  category: "new",
  tags: ["announcements"],
  authors: ["luke", "samuel"],
  date: new Date("2025-07-14"),
});

Organizations on HCB do so much and raise a ton of money through donations! Yet, donors and other people interested in your mission are often left out of the loop.

We've launched <b>Organization Announcements</b> to help you send out announcements to people who want to stay up to date with your organization! This includes donors and anyone else who wants to follow you.

You'll now be able to:
- 👥 Have people follow your organization with the Follow button on your Transactions and Announcements pages
- 📣 Send out announcements using a rich-text editor and blocks that include data from HCB

> - 🗒️ See announcements from organizations that you follow in your email and on the new Feed page

Reviewer: ```suggestion
- 🗒️ See announcements on the new <a href="https://hcb.hackclub.com/my/feed">Feed</a> page, and receive them in your email inbox.
```
